lua/lazyvim/plugins/extras/lang/rust.lua

```diff
@@ -150,13 +122,12 @@ return {
   {
     "nvim-neotest/neotest",
     optional = true,
-    dependencies = {
-      "rouge8/neotest-rust",
-    },
-    opts = {
-      adapters = {
-        ["neotest-rust"] = {},
-      },
-    },
+    opts = function(_, opts)
+      opts.adapters = opts.adapters or {}
+      vim.list_extend(opts.adapters, {
+        require('rustaceanvim.neotest'),
+      })
+    end
```

Comment:
This may want to be moved to the `dependencies` section of `mrcjkb/rustaceanvim`. 

Currently rustaceanvim will only use neotest as it's `test_executor` if the `rustaceanvim.neotest` module was loaded prior to rustaceanvim being configured. (https://github.com/mrcjkb/rustaceanvim/blob/4.7.0/lua/rustaceanvim/config/internal.lua#L42-L48)

Reply:
rustaceanvim loads its configuration lazily (when opening a rust file). So it should still pick up the neotest config as long as neotest isn't lazy loaded by lazy.nvim.